src/reducers/index.js

```diff
@@ -144,9 +144,16 @@ function reducers(state = getInitialState(), action) {
       return newState;
     }
     case ActionTypes.SCROLL_END: {
-      return Object.assign({}, state, {
+      const newState = Object.assign({}, state, {
         scrolling: false,
       });
+      const unchangedRowRange = {
+        firstIndex: state.firstRowIndex,
+        firstOffset: state.firstRowOffset,
+        lastIndex: state.lastIndex,
+        scrollJumpedY: false,
+      };
+      return computeRenderedRows(newState, unchangedRowRange);
```

Comment:
`SCROLL_END` will recompute the rows, and this time since `scrolling` is false, the entire viewport will be updated.

Reply:
see if state.scrollJumpedY can be used instead of false

Reply:
yep, it can be true/false. Doesn't affect buffering